hotkeys: Deactivate control-shift-k and control-shift-s hotkeys.

skunkmb opened the pull request:
This PR fixes #9779.

**Testing Plan:** <!-- How have you tested? -->

Tests are added for ensuring that control-K and control-S fall through.

timabbott commented on static/js/hotkey.js:
```diff
@@ -126,7 +126,7 @@ exports.get_keydown_hotkey = function (e) {
 
     var isCmdOrCtrl = /Mac/i.test(navigator.userAgent) ? e.metaKey : e.ctrlKey;
-    if (isCmdOrCtrl) {
+    if (isCmdOrCtrl && !e.shiftKey) {
```
I think we might want something similar for `keydown_ctrl_mappings` above as well?

skunkmb replied:
@timabbott OK, I've added the same logic to the `keydown_ctrl_mappings` handler, and added a new test. Please let me know if there is more that I should change or fix. Thanks!

skunkmb commented:
@timabbott OK, I've added the same `&& !e.shiftKey` logic to the `keydown_ctrl_mappings` handler based on your review, and added a new test. Please let me know if there is more that I should change or fix. Thanks!

showell commented:
Merged thanks @skunkmb!
